tasks/maven/release.yaml

```diff
@@ -73,6 +73,13 @@ spec:
             mvn versions:set -DnewVersion=$VERSION
 
             mvn --no-transfer-progress clean deploy
+        - image: maven:3.6-openjdk-8-slim
```

Comment:
I don't think we need to do an explicit install step here because the deploy step above will have done it implicitly

Reply:
Oh seems I copied something which shouldn't be there. Will fix